app/views/aspect_memberships/_aspect_membership_dropdown.mobile.haml

```diff
@@ -0,0 +1,13 @@
+- if user_signed_in? && @person != current_user.person
```

Comment:
Related to the last point, the if should probably be around the render for the partial, since it's page specific.